browser/mapping.go

```diff
@@ -677,28 +676,64 @@ func mapBrowserContext(vu moduleVU, bc api.BrowserContext) mapping {
 }
 
 // mapBrowser to the JS module.
-func mapBrowser(vu moduleVU, b api.Browser, wsURL string, isRemoteBrowser bool) mapping {
+func mapBrowser(vu moduleVU, wsURL string, isRemoteBrowser bool) mapping { //nolint:funlen
 	rt := vu.Runtime()
+	bt := chromium.NewBrowserType(vu)
```

Comment:
Question :) Why don't we initialize the browser once in here and then use in the mapped methods below?

Reply:
Thanks for the explanation 🙇